pkg/apis/api/componentreport/types.go

```diff
@@ -290,7 +290,9 @@ type TestDetailsTestStats struct {
 }
 
 type TestDetailsJobStats struct {
-	JobName           string                   `json:"job_name"`
+	NormalizedJobName string                   `json:"job_name"`
```

Comment:
This saves you from having to update the ui but shouldn't it be  `json:"normalized_job_name"`

Reply:
I guess my question would be is it worth renaming at all, it looked like an outlier to me and I suspect someone *might* ask about it later.  That said, I see

```
TriagedIncidents []TriagedIncident     `json:"incidents,omitempty"
ProwJob         string   `bigquery:"prowjob_name"`
```

So there is prior art :smile:

Reply:
True, smells a little weird. We have a lot of things named "ProwJob" and "JobName" and such scattered around the code, I wanted it to be a little more obvious that this particular usage wasn't actually a job name, and is only used as a map key to correlate base and sample jobs. I'll likely rename the json property too in my follow-on UI change.

Reply:
https://github.com/openshift/sippy/pull/2480 gets rid of it entirely